### PR TITLE
BUGFIX: Only show link to accessible nodes

### PR DESCRIPTION
**Pull request by ComiR**
Fixed some misleading text on the listing page and added i18n.

related to #1578.

for 3.x upmerge have a look at https://github.com/ComiR/neos-development-collection/commit/e2d0e44dd6f21179b2e9eeeac1481ae3a4f8a3f6.

### CODE REVIEW

File: TYPO3.Neos/Resources/Private/Templates/Media/Asset/RelatedNodes.html [hunk 1]
```diff
@@ -61,10 +61,12 @@
...
 										<f:if condition="{relatedNode.contextDocumentNode}">
 											<f:then>
-												<neos:link.node node="{relatedNode.contextDocumentNode}" target="_blank" title="{neos:backend.translate(id: 'edit')} \"{neos:backend.translate(id: documentNode.node.label)}\""><i class="icon-external-link"></i></neos:link.node>
+												<f:if condition="{relatedNode.accessible}">
+													<neos:link.node node="{relatedNode.contextDocumentNode}" target="_blank" title="{neos:backend.translate(id: 'workspaces.openPageInWorkspace', source: 'Modules', package: 'TYPO3.Neos', arguments: {0: relatedNode.nodeData.workspace.title})}"><i class="icon-external-link"></i></neos:link.node>
+												</f:if>
 											</f:then>
 											<f:else>
-												<span><i class="icon-warning" data-neos-toggle="tooltip" data-placement="left" title="Broken node, missing document node"></i></span>
+												<span><i class="icon-warning" data-neos-toggle="tooltip" data-placement="left" title="{neos:backend.translate(id: 'media.missingDocumentNode', source: 'Modules')}"></i></span>
```
Comment (aertmann) [inline TYPO3.Neos/Resources/Private/Templates/Media/Asset/RelatedNodes.html]: no package argument

File: TYPO3.Neos/Resources/Private/Templates/Media/Asset/RelatedNodes.html [hunk 2]
```diff
@@ -95,15 +97,15 @@
...
 															</f:then>
 															<f:else>
-																<i class="icon-group" title="{neos:backend.translate(id: 'workspaces.internalWorkspace', source: 'Modules')}" data-neos-toggle="tooltip"></i>
+																<i class="icon-group" title="{neos:backend.translate(id: 'workspaces.internalWorkspace', source: 'Modules', package: 'TYPO3.Neos')}" data-neos-toggle="tooltip"></i>
 															</f:else>
 														</f:if>
 													</f:then>
 													<f:else>
-														<i class="icon-lock" title="{neos:backend.translate(id: 'workspaces.readonlyWorkspace', source: 'Modules')}" data-neos-toggle="tooltip"></i>
+														<i class="icon-lock" title="{neos:backend.translate(id: 'media.noAccessToWorkspace', source: 'Modules', arguments: {0: relatedNode.nodeData.workspace.title})}" data-neos-toggle="tooltip"></i>
```
Comment (aertmann) [inline TYPO3.Neos/Resources/Private/Templates/Media/Asset/RelatedNodes.html]: no package argument

**Comment by kitsunet** [comment]
Thanks! Looks good so far, but I would change the PR title as the commit message title seems misleading.

**Comment by ComiR** [comment]
Change title to what? I know that the majority of changed lines is only adding `, package: 'TYPO3.Neos'` to the translate viewhelper. But there are only two real changes:
1. the two added/changed i18n
2. the reason for this commit: [L64-66](https://github.com/neos/neos-development-collection/pull/1579/files#diff-263e2136be4cdff2fe159e3f0dcb4954R64) - the link to the document node containing the node (not the node itself - how one could maybe interpret the current title) only is shown if the user has access to the node (since a link to a workspace which to user has no access to doesn't make sense).

Since the commit is always for the whole collection, I never know if I should add the package or another hint to where the change was. The commit messages would get so long...

Just give me a good title and I'll change it ;)

**Comment by aertmann** [comment]
title is fine IMO

**Comment by kitsunet** [comment]
Ok, I see, this little line was drowned in the rest of the change and I didn't notice it. 😆 Sorry, all fine 👍

**Pull request merged**